app/tests/spec/views/settings/emails.js

```diff
@@ -97,37 +97,26 @@ define(function (require, exports, module) {
       });
     });
 
-    describe('feature disabled', () => {
-      describe('for user', () => {
+    describe('feature gated in unverified session', () => {
+      beforeEach(() => {
+        sinon.stub(account, 'sessionVerificationStatus').callsFake(() => {
+          return p({sessionVerified: false});
```

Comment:
I'm trying to [prepare for the future using native promises](https://github.com/mozilla/fxa-content-server/pull/5543), in the unit tests, instead of `p(...)`, can you `Promise.resolve`? For the main app, let's still use `p()` until we introduce the polyfill for old browsers.